src/cli/subcommands/attach_cmd.rs

```diff
@@ -299,36 +300,36 @@ impl AttachCommand {
         set_module(context);
 
         // Net API
-        bind_func!(context, token, net_addrs_listen);
-        bind_func!(context, token, net_peers);
+        bind_func!(context, token, |_, t| net_addrs_listen(t));
```

Comment:
You could use a helper function here.
```suggestion
        bind_func!(context, token, nullary(net_addrs_listen));
```

Reply:
As discussed, switching to the unrolling approach to make the diff set small